src/main/java/seedu/address/logic/commands/StatusCommand.java

```diff
@@ -29,7 +29,7 @@
             + "Parameters: INDEX (must be a positive integer) "
             + PREFIX_STATUS + "[STATUS]\n"
             + "Example: " + COMMAND_WORD + " 1 "
```

Comment:
Items in square brackets are supposed to be optional.

Reply:
To clear up any misunderstandings a `Status` is optional -- if left blank, it will simply remove the `Status` from the person at the given `index`, so I will leave it as it is unless there are any changes that need to be made with respect to this logic 😄